src/schema/rules/files/raw/task.yaml

```diff
@@ -29,7 +29,6 @@ timeseries:
     - beh
     - eeg
     - ieeg
-    - meg
```

Comment:
why is meg being removed here?

Reply:
Because it is handled separately below to allow for the proc extension.